collections/sentinel-s2-l2a-mosaic-120.yaml

```diff
@@ -173,18 +173,81 @@ CubeDimensions:
     extent:
       - -180
       - 180
+    reference_system:
+      $schema: https://proj.org/schemas/v0.4/projjson.schema.json
```

Comment:
Can't find where you got this CRS, but since it's more as a placeholder, this comment can be ignored.